app/controllers/saml_idp_controller.rb

```diff
@@ -58,7 +58,9 @@ def render_template_for(message, action_url, type)
   end
 
   def apply_secure_headers_override
-    use_secure_headers_override(:saml)
+    override_content_security_policy_directives(
+      form_action: ["'self'", *SecureHeadersWhitelister.whitelisted_domains]
```

Comment:
Originally I wanted to try to do the domain only for the service provider in question, however, for things like the `/saml/test` endpoint, our specs expect to see *all* domains, so I didn't want to change that at this time.

Maybe we could make sure the `/auth` endpoint only pulls the `acl_url` for the current service provder, and the `/test` endpoint still shows all?